$NetBSD: patch-ac,v 1.3 2013/03/20 07:25:37 obache Exp $

* Not to be unwanted CFLAGS modification. 
* Use pkgsrc's PTHREAD related compiler flags.

--- configure.orig	2012-01-30 08:06:13.000000000 +0000
+++ configure
@@ -18548,8 +18548,6 @@ fi
 if test "${enable_debug+set}" = set; then
   enableval="$enable_debug"
   CFLAGS="$CFLAGS -g3 -O0"
-else
-  CFLAGS="$CFLAGS -g -O2"
 fi;
 
 
@@ -18641,8 +18639,8 @@ else
 fi;
 
 CFLAGS="$CFLAGS -std=gnu99 -Wall -Wextra"
-CPPFLAGS="$CPPFLAGS -I$includedir -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS"
-LDFLAGS="$LDFLAGS -L$libdir -lpthread"
+CPPFLAGS="$CPPFLAGS -I$includedir $PTHREAD_CFLAGS -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS"
+LDFLAGS="$LDFLAGS -L$libdir $PTHREAD_LDFLAGS $PTHREAD_LIBS"
 
 
 # Check whether --with-resolver or --without-resolver was given.
